content/en/docs/ops/configuration/mesh/app-health-check/index.md

```diff
@@ -106,7 +106,7 @@ This approach rewrites the application `PodSpec` readiness/liveness probe, such
 [Pilot agent](/docs/reference/commands/pilot-agent/). Pilot agent then redirects the
 request to application, and strips the response body only returning the response code.
 
-You have two ways to enable Istio to rewrite the liveness HTTP probes.
+This feature is enabled when installing via the Istio Operator and using the `default` profile. If you find that the profile used to install Istio does not have it enabled, you have two ways to enable the rewrite of the liveness HTTP probes:
```

Comment:
Why just when installing via the Istio Operator. If the `default` profile is installed using `istioctl` it will also be enabled, right?

```suggestion
This feature is enabled is Istio is installed using the `default` profile. If the profile used to install Istio does not have it enabled, you have two ways to enable the rewrite of the liveness HTTP probes:
```